### PR TITLE
Update networking test cases

### LINKED ISSUE #5293: Need to fix the vic machine create networking tests
Public network - invalid
Management network -invalid
Bridge network - invalid

Are all broken, they need to be fixed.

### PR DESCRIPTION
**Pull request by morris-jason**
Fixes #5293

### CODE REVIEW

File: tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot [hunk 1]
```diff
@@ -249,21 +241,24 @@ Bridge network - ESX none
...
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
 
-    #Set Test Environment Variables
-    ## Attempt to cleanup old/canceled tests
-    #Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
-    #Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
```
Comment (rogeliosanchez) [inline tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot]: @morris-jason, are lines 251, 252, and 253 repeats of lines 247, 248, and 249?  Is there a reason you want to cleanup old/canceled tests twice?  Curious about this.

File: tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot [hunk 2]
```diff
@@ -249,21 +241,26 @@ Bridge network - ESX none
...
 
-    ## Delete the portgroup added by env vars keyword
-    #Cleanup VCH Bridge Network  %{VCH-NAME}
-    #Cleanup VIC Appliance On Test Server
+    ${output}=  Run  bin/vic-machine-linux create --name=%{VCH-NAME} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=AAAAAAAAAA ${vicmachinetls} --insecure-registry harbor.ci.drone.local
+
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Should Contain  ${output}   Installer completed successfully
+
+    # Attempt to cleanup old/canceled tests
```
Comment (anchal-agrawal) [inline tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot]: This comment doesn't seem accurate.

File: tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot (hunk 1 above)
Comment (morris-jason) [inline tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot]: Yupp, oops, fixed.

File: tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot (hunk 2 above)
Comment (morris-jason) [inline tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot]: Separated the port group removal and added another comment.

**Pull request merged**